dataprofiler/tests/data_readers/test_data_utils.py

```diff
@@ -31,13 +31,25 @@ def test_file_UTF_encoding_detection(self):
                  encoding="utf-16"),
             dict(path=os.path.join(test_dir, 'json/iris-utf-32.json'),
                  encoding="utf-32"),
+            dict(path=os.path.join(test_dir, 'txt/utf8.txt'),
+                 encoding='utf-8'),
+            # Failing Test
+            #dict(path=os.path.join(test_dir, 'csv/zomato.csv'),
+            #     encoding='ISO-8859-1'), 
```

Comment:
This should work, correct?

Reply:
Here were the results I got,
Chardet 20L, 1024B - Windows-1254 => utf-8; Fails to Decode
Chardet Full File - Windows-1254 => utf-8; Fails to Decode
Charset Normalizer 10k - iso8859_10; Decodes but content is different
Charset Normalizer Full File - cp1125; Decodes but content is different

Reply:
Charset Normalizer with 10k samples is very close though. It gets 99.987 % of characters correctly decoded.

Reply:
I could modify the test to check for getting 99.9% of characters, but it would still fail with Chardet (and cause the GitHub actions to fail too) as Charset Normalizer isn't a required package. That's why I just commented out the test and added a comment for now